mender/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: "3.7.4"
+appVersion: "3.7.5"
 description: Mender is a robust and secure way to update all your software and deploy your IoT devices at scale with support for customization
 name: mender
 version: 5.9.0
```

Comment:
When we create a new appVersion release, we also bump the helm chart version:
```suggestion
version: 5.9.1
```

Actually, it deserves a mention in the CHANGELOG: something like this:

```
## Version 5.9.1
* Upgrade to Mender version `3.7.5`
```